tests/unit/test_utils.h

```diff
@@ -55,6 +55,8 @@ inline VecSimIndex *CreateNewIndex(IndexParams &index_params, VecSimType type,
     return VecSimIndex_New(&params);
 }
 
+extern VecsimQueryType query_types[4];
```

Comment:
Why using `extern` instead of defining the `query_types` here?

Reply:
Because it is a global array on `h` file. If included multiple times, there will be multiple definitions of the same array